driver/collector/mysql_collector.py

```diff
@@ -213,28 +203,20 @@ def collect_metrics(self) -> Dict[str, Any]:
         else:
             metrics["global"]["engine"]["master_status"] = ""
 
+        metrics["global"]["performance_schema"][
+            "events_statements_summary_by_digest"
+        ] = json.dumps(self._make_list(*self._cmd(self.QUERY_DIGEST_TIME)))
         if float(self._version) >= 8.0:
             # latency histogram
-            lat_hist_list = []
-            lat_hist = self._cmd(self.METRICS_LATENCY_HIST_SQL)[0]
-            for lat_row in lat_hist:
-                lat_row_new = [
-                    float(elem) if isinstance(elem, Decimal) else elem
-                    for elem in lat_row
-                ]
-                lat_elem = LatencyHistogram._make(lat_row_new)._asdict()
-                lat_hist_list.append(lat_elem)
             metrics["global"]["performance_schema"][
                 "events_statements_histogram_global"
-            ] = json.dumps(lat_hist_list)
+            ] = json.dumps(self._make_list(*self._cmd(self.METRICS_LATENCY_HIST_SQL)))
```

Comment:
nit: consider either unpacking the result of `self._cmd` on a seperate line or even keeping packed but with a descriptive temporary name to make this more readable.